apps/condo/domains/common/components/containers/BehaviorRecorder.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const BehaviorRecorder = () => {
+    return (
+        <div dangerouslySetInnerHTML={{
+            __html: `<script type="text/javascript" defer>
+                var _protocol = (("https:" == document.location.protocol) ? " https://" : " http://");
+                var _site_hash_code = "4b23250e438223063d2fb9af042a8199";
```

Comment:
from env in JSON format like FIREBASE_CONFIG

Reply:
+

Reply:
++